src/development.js

```diff
@@ -22,10 +22,15 @@ export default function catchErrors({ filename, components, imports }) {
         } else {
           console.error(err);
         }
-        return React.createElement(ErrorReporter, {
-          error: err,
-          filename
-        });
+        return React.createElement(ErrorReporter,
```

Comment:
Oh, and it's entirely possible the user doesn't have `Object.assign` polyfill. Please add `babel-plugin-object-assign` to this project's `.babelrc` so it compiles to `_extend` helper call.

You can also set `stage` to be `0` and just say `React.createElement(ErrorReporter, { error: err, filename, ...reporterOptions })`. In fact I'd prefer this.